circle.yml

```diff
@@ -27,6 +27,9 @@ test:
     - yarn test -- --runInBand
 
 deployment:
+  release:
+    branch: master
+      - yarn run semantic-release
```

Comment:
I think there is some legacy stuff around [here](https://github.com/eessex/reaction/blob/0444c1109a60d8afb225c5034b840be83d191e7d/circle.yml#L38) that relates to [legacy deployment](https://github.com/artsy/reaction#deployments) process. Once we've verified that this works we should make sure to remove that and update README.

Reply:
For sure!